recipes-openxt/xenclient/xenclient-installer_git.bb

```diff
@@ -5,7 +5,7 @@ LIC_FILES_CHKSUM = "file://COPYING;md5=4641e94ec96f98fabc56ff9cc48be14b"
 PV = "0+git${SRCPV}"
 
 SRCREV = "${AUTOREV}"
-SRC_URI = "git://${OPENXT_GIT_MIRROR}/installer.git;protocol=${OPENXT_GIT_PROTOCOL};branch=${OPENXT_BRANCH} \
+SRC_URI = "git://github.com/mahantesh-ais/installer.git;protocol=${OPENXT_GIT_PROTOCOL};branch=pr \
```

Comment:
You'll need to drop this... for future reference, you can override this in your local.conf for dev/testing purposes.

Reply:
Ack, will fix it.